static/js/upload.js

```diff
@@ -279,7 +285,7 @@ export function setup_upload(config) {
     });
 
     uppy.on("info-visible", () => {
-        const info = uppy.getState().info;
+        const [info] = uppy.getState().info;
```

Comment:
@IvanToy can you explain why this format change is required -- were we fetching this data structure incorrectly, or something else? And can this list contain more than one thing?

https://uppy.io/docs/uppy/#uppy-setState-patch doesn't suggest to me that this should usually be a list.

Reply:
@timabbott  even though in the uppy documentation, it is not suggested that the state should be a list, the way state is being fetched in this particular case requires the use of a list as it comes as a list. 

**Example of fetching state without the use of list:**
<img width="576" alt="withoutAlist" src="https://user-images.githubusercontent.com/68671067/181049276-1f31b78f-92f4-4da9-b791-96d156a06885.png">

<img width="417" alt="withouAlistConsole" src="https://user-images.githubusercontent.com/68671067/181049322-f638cc31-f8b0-4ef1-9f8b-b2613f1a8fe7.png">

**Example of fetching state as a list containing a dictionary:**
<img width="590" alt="withAlist" src="https://user-images.githubusercontent.com/68671067/181051902-11d43f2b-7b46-4876-a566-d57201a59cf0.png">


<img width="417" alt="withAListConsole" src="https://user-images.githubusercontent.com/68671067/181051102-893a8e13-2675-4d36-bf91-a828dcfcf90e.png">

List contains only one dictionary. Even if you try uploading several files that exceeds allowed size, only the first file being uploaded will be in that dictionary. If you try uploading couple of files and one of them exceeds allowed size, only the file exceeding allowed size will be in that dictionary.

Reply:
OK, we can document this for ourselves in a comment.

Reply:
I've done that and pushed back here.